railway/scripts/test_notification.py

```
@@ -127,43 +76,47 @@ def main():
-    if passed == total:
+    if success_count == total_count:
         print("🎉 全てのテストが成功しました！")
         print("Railway Log Monitorが正常に動作しています。")
-        sys.exit(0)
+        return True
     else:
-        print("⚠️  一部のテストが失敗しました")
+        print("⚠️ 一部のテストが失敗しました。")
         print("設定を確認してください。")
-        sys.exit(1)
+        return False
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)